tests/php/Core/Cache/CacheTest.php

```diff
@@ -18,38 +17,29 @@ protected function setUp()
     {
         parent::setUp();
 
-        Config::modify()
-            ->set(
-                Injector::class,
-                ApcuCacheFactory::class,
-                [
-                    'constructor' => [ 'version' => 4400 ]
-                ]
-            )
-            ->set(
-                Injector::class,
-                CacheInterface::class . '.TestApcuCache',
-                [
+        Injector::inst()
+            ->load([
+                ApcuCacheFactory::class => [
+                    'constructor' => [ 'version' => 'ss40test' ]
```

Comment:
Because apcucache uses version as a cache prefix, and it physically breaks on numeric keys.